src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -207,6 +222,17 @@ public static Test createTest(long startTime, CommandLine commandline, Options o
                 events = 0;
             }
 
+            if (commandline.hasOption("flush")) {
+                int flushEvents = Integer.parseInt(commandline.getOptionValue("flush"));
+                if (flushEvents > 0) {
+                    flushEventsPerProducer = flushEvents;
+                } else {
+                    flushEventsPerProducer = Integer.MAX_VALUE;
```

Comment:
In this case, shouldn't we raise an exception instead of assuming the default value?

Reply:
no,  if the user is not supply the parameter -flush , then it normal case is "flush" after all the writes; so , no exception, but we  set to MAX to indicate that flush will be done after writing all the events.